.github/workflows/ci-tests.yml

```
@@ -0,0 +1,28 @@
+name: CI Tests
```

```suggestion
name: Test tarball builds
```

CI tests is pretty generic (we may add different CI tests later!)